test/karma.conf.js

```diff
@@ -14,7 +14,8 @@ module.exports = function(config) {
       '../build/temp/ie8/videojs-ie8.min.js',
       '../test/globals-shim.js',
       '../test/unit/**/*.js',
-      { pattern: '../src/**/*.js', watched: true, included: false, served: false }
```

Comment:
This was in there because it causes karma to watch the source files for changes (but not serve or run them). I don't believe that was happening otherwise. Was there a need to remove it?